pymc3/gp/gp.py

```diff
@@ -78,7 +57,7 @@ def conditional(self, name, n_points, Xnew, *args, **kwargs):
 
 
 @conditioned_vars(["X", "f"])
-class Latent(GPBase):
+class Latent(Base):
     """ Where the GP f isnt integrated out, and is sampled explicitly
```

Comment:
I think this could be improved with a bit more doc strings. Maybe just with a simple example from somewhere else.